terraform/README.md

```diff
@@ -131,6 +132,14 @@ This will generate the `terraform-configuration.md` markdown file.
 
 ## Troubleshooting
 
+### KeyVault Firewall Error
+
+When running locally you see an error simular to the following:
```

Comment:
sorry, pedantry, but 'similar' not simular

Reply:
I'm so glad somebody is reading the docs, nice catch 😊